stdlib/multiprocessing/popen_fork.pyi

```diff
@@ -0,0 +1,22 @@
+import sys
+
+from .process import BaseProcess
+from .util import Finalize
+
+if sys.platform != "win32":
+    __all__ = ["Popen"]
+
+    class Popen:
+        finalizer: Finalize | None
+        method: str
+        pid: int | None
```

Comment:
Why `| None`? It looks like it's set as the return value of a call to `os.fork()` in the `_launch()` method:  https://github.com/python/cpython/blob/bceb197947bbaebb11e01195bdce4f240fdf9332/Lib/multiprocessing/popen_fork.py#L66

Note that the `_launch()` method is called in the `__init__` method: https://github.com/python/cpython/blob/bceb197947bbaebb11e01195bdce4f240fdf9332/Lib/multiprocessing/popen_fork.py#L19

And the typeshed stub says that `os.fork()` returns an `int`: https://github.com/python/typeshed/blob/177bbf7864b544880e7db8f898d14aa5c3561f07/stdlib/os/__init__.pyi#L821

```suggestion
        pid: int
```

Reply:
This is where I wasn't sure how to properly model what's occurring. There is no guarantee that the attribute will exist, as it's not initialized before that. So assuming the module is used properly, this should be an int, but other circumstances this would raise an attribute error. As a result, I modelled this with a `| None` to capture conceptually that it may not actually exist

Reply:
Sometimes it will exist, sometimes it won't, but whatever the case, it will never be `None`, so `| None` is incorrect. Just do `pid: int`, and add a comment saying that it doesn't always exist. In these situations, we prioritise avoiding false positives over avoiding false negatives.

Reply:
Ah, makes sense. I must have accidentally removed the implementation before that. Added the comment RE: `sentinel`